### DISCUSSION TITLE
noScribe: new App for transcribing interviews, with speaker detection

### DISCUSSION THREAD

**Discussion by kaixxx** [ggml-org/whisper.cpp#940] [Show and tell]
![noScribe App](https://github.com/ggerganov/whisper.cpp/assets/3353272/4f38c2b3-0b61-481d-b504-3be393714682)
 
**>>> [noScribe on GitHub](https://github.com/kaixxx/noScribe)**
 
The main purpose of this app is to transcribe interviews for qualitative research or journalistic use. It uses **whisper.cpp** for transcription and [**pyannote** from Hervé Bredin](https://github.com/pyannote/pyannote-audio) to identify different speakers. It also includes a nice **MS Word-interface** to review, verify and correct the resulting transcript. 

The main goal was to create an easy-to-use interface for my students and other non techie people (like my colleagues from the humanities department ;) ). Take a look, I appreciate any reactions or suggestions! Also: Please spread the word if you know people that might benefit from this app.

For now, the app is running on windows only (installer included). But I would really appreciate any help in bringing this to mac and linux too. It’s all open source (GPL-3.0), programmed in python and prepared for cross-platform support (see [here for more info](https://github.com/kaixxx/noScribe#development-and-contribution).
 
**Thank you, Georgi Gerganov, and all the great folks here for the work you put into whisper.cpp!**

**Comment by caryknoop** [comment]
Very cool!

Will you provide a CUDA enabled build for this or at least describe how we can hook CUDA-enabled Whisper to this?

**Comment by kaixxx** [comment]
There is not much I can do since I don't own a graphics card that supports CUDA. But noScribe uses the compiled "main.exe" from whisper.cpp (in the main app folder) and the large and small model (in the subfolder models). If there are CUDA-enabled versions available of these files, you could simply exchange these. 

Please keep me updated, I am curious to know if this works!
